index.html

```diff
@@ -596,6 +655,20 @@
         return value;
     }
 
+    function toggle_more_stats() {
+        const stats = document.getElementById('helptext__advance_stats');
+        const civs = document.getElementById('helptext__x_ref');
+        const link = document.getElementById('helptext__more_link');
+        if (civs.style.display === 'none') {
+            stats.style.display = 'block';
+            civs.style.display = 'block';
+            link.innerHTML = 'Less Info &#x25B2;';
```

Comment:
```suggestion
            link.textContent = 'Less Info ▲';
```

just fwiw: textContent is preferred to reduce likelihood of XSS.